libs/jump_portrait/src/jump_portrait/fetch.py

```diff
@@ -27,8 +27,9 @@
     get_image_from_s3uri,
     read_parquet_s3,
 )
-from jump_portrait.utils import batch_processing, parallel
-
+from jump_portrait.utils import batch_processing, parallel, try_function
+from typing import List
```

Comment:
This can be replaced by just 'list' since python3.9 IIRC, so no need for the extra import.